message/messaging_service.cc

```diff
@@ -1415,6 +1419,16 @@ future<raft::read_barrier_reply> messaging_service::send_raft_execute_read_barri
    return send_message_timeout<future<raft::read_barrier_reply>>(this, messaging_verb::RAFT_EXECUTE_READ_BARRIER_ON_LEADER, std::move(id), timeout, std::move(gid), std::move(from_id), std::move(dst_id));
 }
 
+void messaging_service::register_forward_request(std::function<future<query::forward_result> (query::forward_request req)>&& func) {
+    register_handler(this, netw::messaging_verb::FORWARD_REQUEST, std::move(func));
+}
+future<> messaging_service::unregister_forward_request() {
+    return unregister_handler(netw::messaging_verb::FORWARD_REQUEST);
+}
+future<query::forward_result> messaging_service::send_forward_request(msg_addr id, query::forward_request req) {
+    return send_message<future<query::forward_result>>(this, messaging_verb::FORWARD_REQUEST, std::move(id), std::move(req));
+}
+
```

Comment:
Since 0ea79559a669259405265194aa33c1b8f7456ecc we can autogenerate this boilerplate, please use it instead.